cmd/Makefile.am

```diff
@@ -263,6 +263,8 @@ snap_confine_snap_confine_SOURCES = \
 	snap-confine/cookie-support.h \
 	snap-confine/mount-support-nvidia.c \
 	snap-confine/mount-support-nvidia.h \
+	snap-confine/mount-support-wsl2-gpu.c \
+	snap-confine/mount-support-wsl2-gpu.h \
```

Comment:
can you add those files to `new_format` list, then run `make fmt` and add commit only the changes to the wsl files?